### ISSUE TITLE
Linking from examples directory fails

### ISSUE THREAD

**Issue by perrin4869** [jspm/jspm-cli#1754]
I'm using jspm in my react component: [react-stay-scrolled](https://github.com/perrin4869/react-stay-scrolled/). I'm trying to link the component from inside the examples directory, however, it fails as follows:

```bash
$ cd examples
$ jspm link ..
Replace installed version? [Yes]: Yes
     Updating registry cache...
     Processing linked package npm:react-stay-scrolled@0.5.1

warn It's taking a long time to process the dependencies of npm:react-stay-scrolled@0.5.1.
     This package may need an ignore property to indicate test or example folders for jspm to skip.
     

<--- Last few GCs --->

   62474 ms: Scavenge 1396.3 (1457.3) -> 1396.3 (1457.3) MB, 1.5 / 0 ms (+ 33.1 ms in 1 steps since last GC) [allocation failure] [incremental marking delaying mark-sweep].
   63324 ms: Mark-sweep 1396.3 (1457.3) -> 1396.1 (1457.3) MB, 850.5 / 0 ms (+ 61.1 ms in 8 steps since start of marking, biggest step 33.1 ms) [last resort gc].
   64158 ms: Mark-sweep 1396.1 (1457.3) -> 1396.1 (1457.3) MB, 833.6 / 0 ms [last resort gc].


<--- JS stacktrace --->

==== JS stack trace =========================================


==== Details ================================================

=====================


FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory
[1]    22336 abort      jspm link ..
```

I added the `files` property to `package.json` but it doesn't seem to help

**Comment by guybedford** [comment]
Sure, I think we may need to refine this files and ignore filter for linking, as its globbing the whole folder otherwise.

**Comment by perrin4869** [comment]
The weird thing is, there are not that many files in that project, you could copy them all in a second...

**Comment by guybedford** [comment]
node_modules is included here. Also the example folder itself potentially.

**Comment by guybedford** [comment]
Fixed in https://github.com/jspm/npm/commit/9f01674e0f3bba88e1f06740a114342fd9755c58.

**Comment by perrin4869** [comment]
Awesome, really looking forward to the release!
Btw out of curiosity, what was the problem here?

**Comment by guybedford** [comment]
The jspm_packages folder itself was being processed.

**Comment by guybedford** [comment]
Released in jspm 0.17.0-beta.15.
